source/shared/cpp/ObjectModel/TextBlock.h

```diff
@@ -50,6 +51,11 @@ class TextBlock : public BaseCardElement
     HorizontalAlignment GetHorizontalAlignment() const;
     void SetHorizontalAlignment(const HorizontalAlignment value);
 
+    void SetLanguage(const std::locale& value);
+
+    std::wstring StringToWstring(const std::string& in) const;
+    std::string WstringToString(const std::wstring& in) const;
```

Comment:
Do these need to be public? Also, since these are generic utility functions and not really part of TextBlock consider moving them to ParseUtil.